spec/unit/appmgmt_spec.rb

```diff
@@ -46,7 +46,7 @@ def compile_to_env_catalog(string)
 
       application app {
         prod { one: host => ahost, export => Cap[cap] }
-        cons { two: host => ahost, consume => Cap[cap] }
```

Comment:
What would the removed test really test though, that isn't tested on the previous or next line?